src/prefect/blocks/core.py

```diff
@@ -232,9 +232,7 @@ def _calculate_schema_checksum(
         block_schema_fields = (
             cls.schema() if block_schema_fields is None else block_schema_fields
         )
-        fields_for_checksum = remove_nested_keys(
-            ["description", "secret_fields"], block_schema_fields
-        )
+        fields_for_checksum = remove_nested_keys(["secret_fields"], block_schema_fields)
```

Comment:
👏

Reply:
Maybe this happens in other places, but should we remove title and example as well?

Reply:
I think we should keep those since they would affect how to form renders in the UI.